File: 프로그래머스/0/120849. 모음 제거/모음 제거.cs
using System;

public class Solution {
    public string solution(string my_string) {
        string answer = my_string;
        string[] arr = new string[5] {"a", "e", "i", "o", "u"};
        
        foreach(var c in arr)
        {
            answer = answer.Replace(c, "");
        }
        return answer;
    }
}

File: 프로그래머스/0/120849. 모음 제거/모음 제거.cpp
#include <string>
#include <vector>
#include <bits/stdc++.h>

using namespace std;

string solution(string my_string) {
    string answer = "";
    string word = "aeiou";
    for(char c : my_string)
    {
        if(word.find(c) == string::npos)
        {
            answer += c;
        }
    }
    return answer;
}